server/spotify-connector.js

```diff
@@ -0,0 +1,34 @@
+require('dotenv').config()
+const request = require('request') // might change to axios
+
+const {CLIENT_ID, CLIENT_SECRET} = process.env,
+      CLIENT_CREDENTIALS = 'client_credentials'
+
+const authOptions = {
+  url: 'https://accounts.spotify.com/api/token',
+  headers: {
+    'Authorization': `Basic ${new Buffer(`${CLIENT_ID}:${CLIENT_SECRET}`).toString('base64')}`
+  },
+  form: {
+    grant_type: CLIENT_CREDENTIALS
+  },
+  json: true
+}
+
+function obtainAccessToken(callback) { request.post(authOptions,  (error, res, body) => {
+  if (!error && res.statusCode === 200) {
+
+    const token = body.access_token
+    callback(null, token)
+  }
+})
+}
+
+function saveAcessToken() {
+  obtainAccessToken((err, res) => {
+    accessToken = res // I don't think it's a great idea to have it here - might move this function to the main file
```

Comment:
Hmm po 1 po co Ci ta zmienna? Możesz bezpośrednio użyć `spotifyApi.setAccessToken(res)`. Po 2 brakuje keyworda `const`. JS nie wywali błędu, bo to JS, ale możesz mieć kiedyś przez to problemy :P

Reply:
smiec zostal, bo wstepnie wygladalo inaczej, juz usunalem